packages/fluentui/accessibility/src/behaviors/Datepicker/datepickerCalendarHeaderBehavior.ts

```diff
@@ -0,0 +1,16 @@
+import { Accessibility } from '../../types';
+/**
+ * @description
+ * Behavior for a datepicked calendar component
+ * @specification
+ * Adds role='group'.
+ */
+export const datepickerCalendarHeaderBehavior: Accessibility<DatepickerCalendarHeaderBehaviorProps> = props => ({
+  attributes: {
+    root: {
+      role: 'group',
```

Comment:
Why header has `group` role? 🤔

Reply:
Placeholder behaviors, I'll be working on those in the next PR :)